doozerlib/metadata.py

```diff
@@ -381,11 +378,34 @@ def latest_build_list(pattern_suffix):
 
                 return refined
 
+            if self.config['is']:
+                if self.config['is'].nvr:
+                    # The image metadata (or, more likely, the currently assembly) has the image
+                    # pinned. Return only the pinned NVR. When a child image is being rebased,
+                    # it uses get_latest_build to find the parent NVR to use (if it is not
+                    # included in the "-i" doozer argument). We need it to find the pinned NVR
+                    # to place in its Dockerfile.
+                    # Pinning also informs gen-payload when attempting to assemble a release.
+                    # strict means raise an exception if not found.
+                    return koji_api.getBuild(self.config['is'].nvr, strict=True)
+                else:
+                    raise IOError(f'"is" definition not yet understood: {self.config["is"]}')
+
             if not assembly:
                 # if assembly is '' (by parameter) or still None after runtime.assembly,
                 # we are returning true latest.
                 builds = latest_build_list('')
             else:
+                basis_event = assembly_basis_event(self.runtime.get_releases_config(), assembly=assembly)
```

Comment:
```suggestion
                basis_event = self.runtime.assembly_basis_event if assembly == self.runtime.assembly else assembly_basis_event(self.runtime.get_releases_config(), assembly=assembly) 
```

Reply:
Is this for performance? Simpler testing? I doesn't seem like it would result in different values.

Reply:
This is unlikely to become a bottleneck, but I spent some time on this to figure out why reloading assembly_basis_event is needed (then I found it is just to support the `assembly` parameter rather than have special reasons). Changing this seems more readable to me, but it doesn't matter much.